mdn/README.md

```diff
@@ -26,4 +27,125 @@ As the `mdn-downloads` bucket already exists and was created outside of Terrafor
 terraform import aws_s3_bucket.mdn-downloads mdn-downloads
 ```
 
-This resource is stored in the Terraform S3 state, so the command doesn't need to be run again.
\ No newline at end of file
+This resource is stored in the Terraform S3 state, so the command doesn't need to be run again.
+
+
+## Restoring a database to a persistent MySQL container
+
+- install kubectl and helm, if necessary
+  - this is done as part of the [stage2.sh](../k8s/install/stage2.sh) install or you can manually run the following:
+
+```sh
+source ../k8s/install/stage2_functions.sh
+export HELM_VERSION=2.2.3  # normally set in stage2.sh
+install_helm
+install_kubectl
+kubectl config # or export KUBECONFIG=~/path/to/kubeconfig
+kubectl get nodes # or another command to verify successful auth
+helm init
+```
+
+- clone mdn/helm-charts
+
+```sh
+git clone https://github.com/mdn/helm-charts charts # TODO: consolidate charts into this repo
+```
+
+- edit charts/mysql/values.yaml
+
+```yaml
+persistence:
+  enabled: true
+  accessMode: ReadWriteOnce
+  size: 40Gi
+
+```
+
+- install from the local directory
+
+```sh
+cd mdn/charts/mysql
```

Comment:
I should have caught this when we were pairing, sorry, but this should be ``cd charts/mysql`` since we're assuming that we're within the same directory as this ``README.md`` file.